frappe/utils/backups.py

```diff
@@ -559,7 +559,7 @@ def new_backup(
 	delete_temp_backups()
 	odb = BackupGenerator(
 		frappe.conf.db_name,
-		frappe.conf.db_name,
+		frappe.conf.db_user or frappe.conf.db_name,
```

Comment:
Why is fallback required here?

Reply:
There was one case where it wasn't getting set here